trace-sdk/src/main/java/io/bitrise/trace/data/dto/FragmentData.java

```diff
@@ -117,4 +118,16 @@ public String toString() {
                 ", spanId='" + spanId + '\'' +
                 '}';
     }
+
+    @Override
+    public boolean equals(@Nullable final Object obj) {
```

Comment:
added a bunch of the equal operators to a few data classes, as the unit tests were giving false negatives if i asked to assert if two objects were equals.